package com.github.caijh.commons.util.exception

class EncryptException(cause: Throwable?) : RuntimeException(cause)
